src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -146,6 +147,77 @@ public void end(boolean interrupted) {
         };
     }
 
+    public CommandBase setAbsoluteAngle(final Rotation2d angle, final double speed) {
+
+        Drive thisDrive = this;
+
+        return new CommandBase() {
+            {
+                addRequirements(thisDrive);
+                setName("Absolute angle");
+            }
+            private double speed2 = speed;
```

Comment:
I would prefer to change the argument name to `aSpeed` instead of making this `speed2`